src/main/generated/resources/assets/minestuck/lang/en_us.json

```diff
@@ -944,6 +944,255 @@
+  "echeladder.rung.45.blood": "Professor 2012",
+  "echeladder.rung.46.blood": "Blocky Tarantino",
+  "echeladder.rung.47.blood": "Protector of Planets",
+  "echeladder.rung.48.blood": "Goreserker Godspawn",
+  "echeladder.rung.49.blood": "Sanguine Unbound",
+  "echeladder.rung.5.blood": "Juveblood Newvein",
+  "echeladder.rung.6.blood": "Cyclecell Sweetkid",
+  "echeladder.rung.7.blood": "Champ-Fry",
+  "echeladder.rung.8.blood": "Scamper Urchin",
+  "echeladder.rung.9.blood": "Render Pains Zazzerbust",
+  "echeladder.rung.0.heart": "Noobling",
+  "echeladder.rung.1.heart": "Rumpchump",
+  "echeladder.rung.10.heart": "14 Pumpchump",
+  "echeladder.rung.11.heart": "Ab-lad-tion",
+  "echeladder.rung.12.heart": "Aerobikid",
+  "echeladder.rung.13.heart": "Sir Aneur of The Rysm Valley",
+  "echeladder.rung.14.heart": "Alchemelad",
+  "echeladder.rung.15.heart": "Benchcrafter Tenderbum",
+  "echeladder.rung.16.heart": "Kneehigh Plumptike",
+  "echeladder.rung.17.heart": "Terratomatychoon",
+  "echeladder.rung.18.heart": "Rung Blaster",
+  "echeladder.rung.19.heart": "Sip Simp",
+  "echeladder.rung.2.heart": "Anklebiter",
+  "echeladder.rung.20.heart": "Kid Arrhythmia",
+  "echeladder.rung.21.heart": "Anti Coagulad",
+  "echeladder.rung.22.heart": "Anklesmiter",
+  "echeladder.rung.23.heart": "Is It A Bird?",
+  "echeladder.rung.24.heart": "Is It A Plane?",
+  "echeladder.rung.25.heart": "It’s Atriu-Man!",
+  "echeladder.rung.26.heart": "Bardycardiabrat",
+  "echeladder.rung.27.heart": "Capilout First Class",
+  "echeladder.rung.28.heart": "Mc Sneezy",
+  "echeladder.rung.29.heart": "Ectocardiolobabysitter",
+  "echeladder.rung.3.heart": "Baby Bechart",
+  "echeladder.rung.30.heart": "Soul Sleuth",
+  "echeladder.rung.31.heart": "Spirit Debutante",
+  "echeladder.rung.32.heart": "Lil Cal’ories",
+  "echeladder.rung.33.heart": "eGo Getter",
+  "echeladder.rung.34.heart": "Animusimp",
+  "echeladder.rung.35.heart": "Adorant Vitalad",
+  "echeladder.rung.36.heart": "Shredder Of The Mortal Coil",
+  "echeladder.rung.37.heart": "Warrant Officer Capilout",
+  "echeladder.rung.38.heart": "Corque Ma-damn",
+  "echeladder.rung.39.heart": "Ponder Orbson",
+  "echeladder.rung.4.heart": "Pain Private",
+  "echeladder.rung.40.heart": "Pro Flyter",
+  "echeladder.rung.41.heart": "The Souls Beckon",
+  "echeladder.rung.42.heart": "Professor Ragnaroc",
+  "echeladder.rung.43.heart": "God Slayer Supreme",
+  "echeladder.rung.44.heart": "Ultra Ego",
+  "echeladder.rung.45.heart": "Soulsand Tycoon",
+  "echeladder.rung.46.heart": "The Final Beats",
+  "echeladder.rung.47.heart": "Center Of The Universe",
+  "echeladder.rung.48.heart": "Man-Skipper Soulsnatch",
+  "echeladder.rung.49.heart": "Master Of Souls",
+  "echeladder.rung.5.heart": "Kidskipper Soulsnatch",
+  "echeladder.rung.6.heart": "Plucky Cadet",
+  "echeladder.rung.7.heart": "Juvesprout",
+  "echeladder.rung.8.heart": "Soulspring",
+  "echeladder.rung.9.heart": "Runghop Chumtot",
```

Comment:
Its been mentioned elsewhere before, but this need to be put in MinestuckEnUsLanguageProvider. Otherwise the instant someone uses runData it will be written over. But again this is more than a minor change so it should not be grouped together with the grist gutter. You would be able to use runData in that case to wipe it out if this is unintentional